apps/experiments/views/experiment.py

```diff
@@ -392,7 +399,16 @@ class DeleteFileFromExperiment(BaseDeleteFileView):
 @permission_required("experiments.view_experiment", raise_exception=True)
 def single_experiment_home(request, team_slug: str, experiment_id: int):
     experiment = get_object_or_404(Experiment, id=experiment_id, team=request.team)
-    user_sessions = ExperimentSession.objects.filter(
+    last_message_created_at = (
+        ChatMessage.objects.filter(
+            chat__experiment_session=OuterRef("pk"),
+        )
+        .order_by("-created_at")
+        .values("created_at")[:1]
+    )
+    user_sessions = ExperimentSession.objects.annotate(
+        last_message_created_at=Subquery(last_message_created_at)
+    ).filter(
```

Comment:
nit: maybe worth making this a manager method to avoid the copy paste

Reply:
Cool. Will throw this in with a followup PR. There are some other housekeeping things I want to do